.ci/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+apt-get -qq update
+apt-get -qq -y install curl
+
+curl -o /usr/bin/jq http://stedolan.github.io/jq/download/linux64/jq && chmod +x /usr/bin/jq
+
+version=$(curl -s https://api.github.com/repos/praekeltfoundation/rapidpro/tags  | jq -r '.[0].name')
+
+echo "$version.$VERSION_SUFFIX" > image-params/tag
```

Comment:
We probably want `"$version$VERSION_SUFFIX"` so we can support an empty suffix. Then we either put the separator in the configured suffix or we do something like `VERSION_SUFFIX=${VERSION_SUFFIX+.$VERSION_SUFFIX}` to put the separator in.